examples/ssx-test-onboard-react-dapp/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
We can add here more context about the example and the instructions to populate the .env and run the application

Reply:
Done